### PR TITLE
UI-23 Design updates for Access V1

### PR DESCRIPTION
**Pull request by julianjuko**

### CODE REVIEW

File: src/Spinner/Spinner.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+interface IProps {
+  className?: string
+  size?: SizeProp
+  color?: string
+}
+
+const Spinner: React.FC<IProps> = ({className, size, color}): JSX.Element => {
+  return (
+    <FontAwesomeIcon
+      className={className || 'fa-spin'}
```
Comment (MichaelDorkhom) [inline src/Spinner/Spinner.tsx:17]: Can we use `defaultProps` on this component to set these default values?

File: src/SmartList/SmartList.tsx [hunk 2]
```diff
@@ -33,10 +34,10 @@ export interface IRowElement {
   className?: string
 }
 
-const SmartList: React.FC<IProps> = ({ data, cols, textIfEmpty, header = true }) => {
+const SmartList: React.FC<IProps> = ({ data, cols, textIfEmpty, loading, header = true }) => {
   const headerContent = (
     <Header
-      cols={cols}
+      cols={loading ? [] : cols}
```
Comment (MichaelDorkhom) [inline src/SmartList/SmartList.tsx:40]: Shouldn't the column names be shown while it's loading?

Comment (julianjuko) [reply src/SmartList/SmartList.tsx:40]: Callum and I decided to have the column names not display while it loads

File: src/SmartList/SmartList.tsx [hunk 3]
```diff
@@ -53,20 +54,36 @@ const SmartList: React.FC<IProps> = ({ data, cols, textIfEmpty, header = true })
...
+          <Col className='list-placeholder'></Col>
+        </Row>
+      </div>
+    }
+
+    if (data.length) {
+      return <div>{listContent}</div>
+    }
+
+    if (textIfEmpty) {
```
Comment (MichaelDorkhom) [inline src/SmartList/SmartList.tsx:70]: Can you add a default value for `textIfEmpty`. Something generic like "No data to display"

Comment (julianjuko) [reply src/SmartList/SmartList.tsx:70]: Actually, we need the option for the SmartList to have a completely null state due to the way we use it in the Subscriptions display in the Hospitals show page of Access, with the option to "Show Expired Plans".

See https://github.com/doseme/admin-dashboard/blob/develop/src/scenes/Dashboard/HospitalsPage/Show/Show.tsx#L180

**Pull request merged**